src/gc/gcinterface.h

```diff
@@ -720,6 +720,9 @@ class IGCHeap {
     // Tells the GC when the VM is suspending threads.
     virtual void SetSuspensionPending(bool fSuspensionPending) = 0;
 
+    // Tells the GC how many YieldProcessor calls are equal to one normalized yield processor call.
+    virtual void SetYieldProcessorNormalizationInfo(uint32_t yieldProcessorNormalizationInfo) = 0;
```

Comment:
This `YieldProcessorNormalizationInfo` does not match what we call YieldProcessorNormalizationInfo in the VM (it is 3 numbers in the VM), and it is does not really tell what this is.

`YieldProcessorScalingFactor` would be a better name for this, I think.

Reply:
SetYieldProcessorScalingFactor sounds good to me